Add contents into the website

Liubuntu opened the pull request:
Hi @PeteHaitch , 

I have just added some contents into the now alive webspage of biocasia2020.bioconductor.org, based on the Bioc2020 materials with modifications to fit BiocAsia2020. Local tests are successful. The contents are not final and up to the committee's approval. The point is to have some common information available early on the website. 

Please review the changes. Some contents still lacking some specific information (e.g., past BiocAsia conference links and hosting cities). We can talk about the details in our meeting next week. Thanks!

Best,
Qian

PeteHaitch commented:
Looks good to me and it'll be great to have some content live!
I'll have to go digging for the previous BioCAsia meeting websites - I feel like each year we might have simply overwritten the previous year's 😬
One small thing: is the use of "donor" to describe the sponsors taken from the BioC meeting? It sounds a little odd to me, I think "sponsor" is better.

Liubuntu commented:
Good catch! Those are copied from the Bioc2020 main website. And I agree that 'sponsor' is much better.

Liubuntu commented:
Hi Peter, @PeteHaitch I have just changed the "donors" into "sponsors", and also added some temporary contents ```coming soon...``` for those empty files to avoid error when clicking on those links.
